node/sign_node_package/action.yml

```diff
@@ -55,10 +55,9 @@ runs:
       if: ${{ inputs.sign_native == 'true' }}
       shell: bash
       run: |
-        FILENAMES="build-*/**/*.tar.gz"
-        if [[ $FILENAMES =~ '*' ]]; then
-          FILENAMES=$(ls $FILENAMES | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/ /g')
-        fi
+        # all prebuilds
+        FILENAMES=$(find build-* -type f -name '*.tar.gz')
+        FILENAMES=$(echo "$FILENAMES)
```

Comment:
Do you need this line? It looks like the quotes aren't matching here if you do

Reply:
I did miss a quote here.  the only way to test is to push changes here, and then re-run actions in other repos (ex: https://github.com/mongodb-js/zstd/actions/runs/12240392950).  Sorry if you're getting notifications

I do need this line, or something like it, because our SSDLC tooling expects space separated filenames and find outputs newline separated filenames.